### PR TITLE
Move the changing of the policy of addon ServiceAccounts

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: ansible/istio/tasks/change_scc.yml [hunk 1]
```diff
@@ -6,8 +6,4 @@
     - istio-egress-service-account
     - istio-pilot-service-account
     - istio-mixer-service-account
-    - istio-ca-service-account
```
Comment (cmoulliard) [inline ansible/istio/tasks/change_scc.yml]: Why do we have to remoe "istio-ca-service-account" @geoand ?

Comment (geoand) [reply ansible/istio/tasks/change_scc.yml]: We don't actually... My local code has it. I'm checking to see why it appears to be deleted

Comment (cmoulliard) [reply ansible/istio/tasks/change_scc.yml]: I think that it was part of a previous distro of istio 0.3.0 or 0.2.12 ....

Comment (geoand) [reply ansible/istio/tasks/change_scc.yml]: https://github.com/geoand/istio-integration/blob/scc-updates/ansible/istio/tasks/change_scc.yml

Comment (geoand) [reply ansible/istio/tasks/change_scc.yml]: It looks like a bug in GitHubs's UI

Comment (geoand) [reply ansible/istio/tasks/change_scc.yml]: I definitely haven't deleted it :)

**Pull request merged**